virtualcluster/pkg/syncer/resources/pod/dws.go

```diff
@@ -64,6 +64,14 @@ func (c *controller) Reconcile(request reconciler.Request) (res reconciler.Resul
 	if err := c.MultiClusterController.Get(request.ClusterName, request.Namespace, request.Name, vPod); err != nil && !apierrors.IsNotFound(err) {
 		return reconciler.Result{Requeue: true}, err
 	}
+	if featuregate.DefaultFeatureGate.Enabled(featuregate.TenantScheduler) {
+		// if LabelTenantSchedulerName is specified, bypass syncing
+		schedulerlabel, ok := vPod.GetLabels()[constants.LabelTenantSchedulerName]
+		if ok && schedulerlabel != "" {
```

Comment:
you can omit ok, with just `if vPod.GetLabels()[constants.LabelTenantSchedulerName] != ""` as it is "" if not found

Reply:
Yes true, but I prefer to play the safe ...